### ISSUE TITLE
PHP: Missing node_modules folder

### ISSUE THREAD

**Issue by stevenscg** [snyk/actions#2]
When I run the php action, I get an error message about a `node_modules` folder as shown below. This project does not use NodeJS.

Expected behavior is that the action will run against the PHP application code in the repository contents checkout and report any findings.

From the log:
```
Run snyk/actions/php@master
/usr/bin/docker run --name snyksnykphp_95690b --label 2e4e3a --workdir /github/workspace --rm -e IMAGE_TAG -e SNYK_TOKEN -e INPUT_ARGS -e INPUT_COMMAND -e HOME -e GITHUB_REF -e GITHUB_SHA -e GITHUB_REPOSITORY -e GITHUB_ACTOR -e GITHUB_WORKFLOW -e GITHUB_HEAD_REF -e GITHUB_BASE_REF -e GITHUB_EVENT_NAME -e GITHUB_WORKSPACE -e GITHUB_ACTION -e GITHUB_EVENT_PATH -e RUNNER_OS -e RUNNER_TOOL_CACHE -e RUNNER_TEMP -e RUNNER_WORKSPACE -e ACTIONS_RUNTIME_URL -e ACTIONS_RUNTIME_TOKEN -e GITHUB_ACTIONS=true -v "/var/run/docker.sock":"/var/run/docker.sock" -v "/home/runner/work/_temp/_github_home":"/github/home" -v "/home/runner/work/_temp/_github_workflow":"/github/workflow" -v "/home/runner/work/myapp/myapp":"/github/workspace" snyk/snyk:php  "snyk" "test" "--severity-threshold=high"
Missing node_modules folder: we can't test without dependencies.
Please run 'npm install' first.

##[error]Docker run failed with exit code 2
```

main.yml:
```yaml
name: Continuous Integration

on: [push]

jobs:
  build:
    runs-on: ubuntu-latest
    steps:
      - name: Checkout repository contents
        uses: actions/checkout@v1

      - name: Run Snyk test - php
        uses: snyk/actions/php@master
        env:
          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
        with:
         args: --severity-threshold=high
```

**Comment by garethr** [comment]
It looks like it's not picking up the `composer.lock` and `composer.json` files.

Could you test out locally using the following:

```
docker run --rm -it --env SNYK_TOKEN -v $(pwd):/app snyk/snyk:php
```

Also if you could run `ls`?

DO you perhaps not have the two Composer files in the root of the application?

**Comment by stevenscg** [comment]
Closing this since we determined that there was no issue with the snyk action.

What happened in this particular case is the project contained a `package.json` file that was not actively used by the production build process. The `node_modules` folder was not present during the build and was correctly detected by the snyk action.

To avoid this issue, we changed the `args` value in `main.yml` above to the following:
```
--severity-threshold=high --file=composer.lock
```

This targets the composer dependencies and ignores other package manager files entirely.

Thanks to Snyk Support for digging in and offering suggestions to troubleshoot the issue.
